avaframe/com1DFAPy/DFAfunctionsCython.pyx

```diff
@@ -693,40 +698,26 @@ def updateFieldsC(cfg, particles, dem, fields):
     # find coordinates in normalized ref (origin (0,0) and cellsize 1)
     # find coordinates of the 4 nearest cornes on the raster
     # prepare for bilinear interpolation
-    Lx0, Lx1, Ly0, Ly1, f00, f10, f01, f11 = getWeights(x, y, csz, interpOption)
-
+    getWeights(x, y, csz, interpOption, Lxy, w)
```

Comment:
I think we should keep the clear distinction of what is input and what is output! So something like 
`l, f = getWeights(...)`

Reply:
how can I make C return two lists if 4 doubles? I did not find out how...